package.json

```diff
@@ -76,7 +76,7 @@
     "yeoman-generator": "^5.0.1"
   },
   "engines": {
-    "node": "10.* || 12.* || >= 14.*"
+    "node": ">= 10.25.* < 13.* || 12.* < 13.* || >= 14.*"
```

Comment:
```suggestion
    "node": ">= 10.25 < 11 || 12.* || >= 14.*"
```

11 and 13 both need to be excluded.

Reply:
You're right, @bmish. For some reason the semver validator I was using was giving me odd results. This makes sense.